### PR TITLE
Adding a simple tag env.

### PR DESCRIPTION
**Pull request by DQN-Labs**
Ignore the deletion of the Srsly2 directory, I had to delete it and add it again after naming it to TagSimple. I hope you guys consider this to be added to the example envs listed in the repo. Thanks.

### CODE REVIEW

**Comment by Ivan-267** [comment]
Hello, first I'd like to thank you for contributing your environment to the examples!

I have a few questions based upon a quick try:
1) Would you be able to include a trained .onnx file in the environment? This can help make testing easier so we try to include it in examples where it is possible to do so (some examples might not have it if onnx export isn't yet implemented for the features the env uses).

2) Might the env from the repository be missing some resource files? I see there's a texture, but it isn't being displayed on my PC using Godot 4.3 so both of the agents and the floor have the same color:

env

If you did have some textures in the local version, it's possible they might have been excluded after uploading to github due to the repository .gitignore or such.

**Comment by Ivan-267** [comment]
Thanks for addressing the points! 

Regarding onnx, if you can get a basic agent that does the task somewhat successfully, you can include it, it does not have be a perfectly trained model (which can be difficult to do for many envs).

We also have a [TestExamples](https://github.com/edbeeching/godot_rl_agents_examples/tree/main/examples/TestExamples) folder which I intended to use for simpler examples or testing examples that don't need to be fully featured, the current description of the folder in readme is:

> This folder contains examples which are mainly made for testing and experimentation, they may not be as stable as the main examples.

Do you think that folder might be suitable for this example?

As another point, regarding the textures, if they are not public domain (or in case you'd like to even if they are), could you include somewhere a small txt file that mentions the source and license of the textures?

Feel free to leave a comment when the PR is ready, I'll ask @edbeeching to do the final review as the repository owner.

**Comment by DQN-Labs** [comment]
Sorry about the stopping and reopening, accident... The textures are MIT licensed, so that shouldn't be a problem. Do you still need me to add the MIT license txt file?

Also, I'm gonna try to train the agent soon to make an onnx file for the env. I'm a little busy right now (school) so I think you can expect it by sometime this weekend.

I think the env is suitable for testexamples, but I'd prefer it to be in the main and test examples. Ultimately, that decision is up to you and @edbeeching.

**Comment by Ivan-267** [comment]
No worries about the reopening. 

Yes, the textures should be compatible, I would still suggest adding a note about the texture author/license somewhere in the project folder to attribute the author, a simple texture license txt file or similar.

No need to hurry at all on the training/onnx, training an env can take a while sometimes. Just leave a comment when the PR is ready.

**Comment by DQN-Labs** [comment]
Sure thing, I've added the license.txt file into the directory. I was actually wrong, it's not an MIT license like I expected, it's a Creative Commons license, but there shouldn't be any issues. Let me know if there's any legal issues, and I can change the textures to something else or make textures myself. I'll get back to you after committing the onnx model. Thanks!

P.S. I'm fine with this example being in the list of TestExamples, it seems perfect for that rather than the normal examples. It just doesn't seem very polished, as the examples there are supposed to showcase the capabilities of the library, which this environment doesn't do very well.

**Comment by DQN-Labs** [comment]
@Ivan-267 @edbeeching The PR is ready for review. I have also added the onnx model as requested.

**Comment by edbeeching** [comment]
Hi @DQN-Labs , thanks for you addition of this example. It seems the texture are still broken, I will leave a few comments for improvements on the PR.

File: examples/TagSimple/AIController3D.gd [hunk 1]
```diff
@@ -0,0 +1,36 @@
+extends AIController3D
+
+var move = Vector2.ZERO
+var jump = false
+@onready var red = $".."
```
Comment (edbeeching) [inline examples/TagSimple/AIController3D.gd:5]: I think if you remove the onready for red and green add `@export Node3D myself` and `@export Node3D other_player` you can combine the two AIController3D scripts into one script.

File: examples/TagSimple/Red.gd [hunk 2]
```diff
@@ -0,0 +1,31 @@
+extends CharacterBody3D
```
Comment (edbeeching) [inline examples/TagSimple/Red.gd:1]: The same comment about the Red.gd and Green.gd scripts, these contain the same code so can the be combined and shared?

**Comment by edbeeching** [comment]
@Ivan-267 I tried to run inference on the onnx model but I had an invalid index error, did you manage to run the onnx model?

**Comment by DQN-Labs** [comment]
Hey, thanks for reviewing the PR @edbeeching @Ivan-267. I'm very sorry for all of these problems. I'm very sorry but I'm really busy with other projects and school so I can't ever find time to come back to all this. I'm closing the PR for now, maybe I can work on it again some later time. Thanks for your amazing dedication to this PR. Much appreciated.

**Pull request closed without merging**